src/app/shared/menu/date/custom-date-range-dropdown.component.ts

```diff
@@ -1,48 +1,87 @@
 import { Component, Input, Output, EventEmitter, OnChanges } from '@angular/core';
 import { Store } from '@ngrx/store';
-import { FilterModel, INTERVAL_HOURLY } from '../../../ngrx';
+import { FilterModel, INTERVAL_HOURLY, IntervalModel, IntervalList } from '../../../ngrx';
 import * as dateUtil from '../../util/date';
 import { GoogleAnalyticsEventAction } from '../../../ngrx/actions';
 
 @Component({
   selector: 'metrics-custom-date-range-dropdown',
   template: `
-    <div class="custom-date-range-dropdown" [class.open]="open">
+    <div class="dropdown" [class.open]="open">
```

Comment:
I havent made this dropdown structure a dropdown component with projected content because adding the custom styles for the dropdown-button and dropdown-content seems like it would just get unnecessarily complicated.

Reply:
Like you'd have to use a bunch of deep selectors or something?

Reply:
No, like I'd want to style the outer components according to what I'm putting in there, not even deep selectors will solve that one